recipes/isa-l/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "cci.20230818":
```

Comment:
why adding this version? Is there a request for it?

Reply:
No, but the latest release was 3 years ago. Tbh, I don't remember any more details about why I chose to add it.

Reply:
I'd rather leave it until someone requests an updated version in an issue or similar to avoid creating kind of artificial versions

Reply:
Ok, I agree. Will revert this.

Reply:
Thank you! 😄